package.json

```diff
@@ -422,6 +422,24 @@
       "isFramework": false,
       "integrity": "9944EBD6EE06BD595BCADD3057CD9BEF4105C3A3952DAE03E54F3114E2E6661F"
     },
+    {
+      "id": "RoslynCopilotLanguageServer",
```

Comment:
nit - a better name is probably 'RoslynCopilot' - including language server makes me think its a separate language server , when really its just an extra dll loaded in the existing language server.